streams/src/main/java/org/apache/kafka/streams/processor/internals/ChangelogRegister.java

```diff
@@ -31,6 +33,8 @@ public interface ChangelogRegister {
      */
     void register(final TopicPartition partition, final ProcessorStateManager stateManager);
 
+    void register(final Set<TopicPartition> partition, final ProcessorStateManager stateManager);
```

Comment:
This is not strictly needed, but I thought it makes registering and unregistering a bit more symmetric.